xapian-applications/omega/omindexcheck.cc

```diff
@@ -40,7 +40,8 @@ unordered_map<string, testcase> tests;
 static void
 index_test(void)
```

Comment:
ISO C uses `(void)` for an empty parameter list.  That also works in C++, but it's more natural to just write `()` in C++ code when there are no parameters (in C `()` means an unspecified number of parameters, for compatibility with pre-standardised C).

Reply:
Okay, I am going to change it